### ISSUE TITLE
Simplify way to find location of individuals

### ISSUE THREAD

**Issue by rmm-fcul** [larics/assisi-playground#52]
An easier way to find the location of an individual, e.g. by sim.Control, would be valuable for computing global statistics while a run is happening.  This is for an interactive exploration of experiments, and might be able to provide direct, within-run feedback; in a similar way to how tracking software would.

Currently, I have to connect a behaviour handler to each, to run get_true_pos()
- this does not seem right right for a monitor function (for instance, these handlers could in principle change the behaviour, and not just observe.)
- it is also a pain to have to keep track of what bees are in the simulation manually (see *forthcoming issue to be posted*)

**Comment by rmm-fcul** [comment]
Damjan, have you had a chance to look at this yet? We have some experiments planned that would benefit from this feature.

**Comment by dmiklic** [comment]
My schedule is a bit packed, but I can make this a priority. When do you plan to do the experiments?

**Comment by rmm-fcul** [comment]
Yes, can we put this at the top of the dev list? We are still discussing the form of experiments but I anticipate putting time in during the next 2-4 weeks.

**Comment by dmiklic** [comment]
I'm aiming to have this working by next monday (october 26). Is this ok?

**Comment by rmm-fcul** [comment]
Sounds good to me!

**Comment by dmiklic** [comment]
I'm running a bit late, but working on it!
